pages/index.tsx

```diff
@@ -48,69 +52,90 @@ export default function Home() {
     const scores = data as PerspectiveScores;
     setScores(scores);
     formatForSentencesAnalysis(scores);
```

Comment:
Don’t call `formatForSentenceAnalysis` in `updateScore`, should be called in a `useEffect`.